docs/content/concepts/graphql-rpc/pagination.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Pagination
+---
+
+{@include: ../../snippets/info-graphql-release.mdx}
+
+
+## Multiple data in one query
+
+The previous RPC service required multiple queries for more complex data retrieval. Now, with GraphQL, you can specify the data you need in one single query.
+
+For example, the following query retrieves the first 20 transaction blocks (along with the digest, the sender's address, the gas object used to pay for the transaction, the gas price, and the gas budget) after a specific transaction block at epoch `97`. In the previous RPC, this would have required multiple API calls.
```

Comment:
```suggestion
For example, the following query retrieves the first 10 transaction blocks (along with the digest, the sender's address, the gas object used to pay for the transaction, the gas price, and the gas budget) after a specific transaction block at epoch `97`. In the previous RPC, this would have required multiple API calls.
```